### PR TITLE
Update html/ from monorail results

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See #798 for methodology

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
@foolip - PTAL.

I did my own first pass using https://docs.google.com/spreadsheets/d/1AJFl3gLfVFjOXRAir9g2BsesdF3fGp9LJHnDtWQLW-c/edit#gid=0 to look for meta bugs etc. Feel free to review my comments and the second commit if you want to validate them at all.

**Comment by foolip** [comment]
Thanks @stephenmcgruer! I'll just do some spot checking of what still remains in the PR.

File: html/canvas/element/text-styles/META.yml [hunk 1]
```diff
@@ -22,3 +22,13 @@ links:
...
+    - product: chrome
+      url: https://crbug.com/1066953
+      results:
+        - test: 2d.text.draw.baseline.hanging.html
+        - test: 2d.text.draw.baseline.ideographic.html
+        - test: 2d.text.draw.space.collapse.end.html
+        - test: 2d.text.draw.space.collapse.other.html
+        - test: 2d.text.draw.space.collapse.space.html
+        - test: 2d.text.draw.space.collapse.start.html
+        - test: 2d.text.measure.width.space.html
```
Comment (foolip) [inline html/canvas/element/text-styles/META.yml:34]: The bug is about this one. The other tests were updated, but it sounds like it was a problem with the tests, fonts not loading. Have you looked at the failure mode of the other tests to see that they also seem to be about whitespace collapsing?

File: html/canvas/offscreen/text/META.yml [hunk 2]
```diff
@@ -4,6 +4,12 @@ links:
       results:
         - test: 2d.text.measure.width.space.html
         - test: 2d.text.measure.width.space.worker.html
+        - test: 2d.text.draw.baseline.hanging.html
```
Comment (foolip) [inline html/canvas/offscreen/text/META.yml:7]: This is https://crbug.com/1066953 again, and this and the next test are named in a way that doesn't obviously match collapsing.

File: html/semantics/embedded-content/media-elements/loading-the-media-resource/META.yml [hunk 3]
```diff
@@ -23,3 +23,20 @@ links:
       results:
         - test: resource-selection-invoke-insert-into-iframe.html
           status: FAIL
+    - product: chrome
+      url: https://crbug.com/593289
```
Comment (foolip) [inline html/semantics/embedded-content/media-elements/loading-the-media-resource/META.yml:27]: Oh look, my old bug :)

**Review by foolip — COMMENTED** [review]
I've reviewed especially the bugs with with many tests linked to them, and spot checked some of the rest. Overall looks great!

File: html/canvas/element/text-styles/META.yml (hunk 1 above)
Comment (foolip) [reply html/canvas/element/text-styles/META.yml:34]: Update: I'll check a few of them.

File: html/canvas/element/text-styles/META.yml [hunk 4]
```diff
@@ -22,3 +22,13 @@ links:
           subtest: System fonts must be computed to explicit values
         - test: 2d.text.font.parse.complex.html
           subtest: 'Canvas test: 2d.text.font.parse.complex'
+    - product: chrome
+      url: https://crbug.com/1066953
+      results:
+        - test: 2d.text.draw.baseline.hanging.html
```
Comment (foolip) [inline html/canvas/element/text-styles/META.yml:28]: Judging by https://wpt.live/html/canvas/element/text-styles/2d.text.draw.baseline.hanging.html and https://wpt.live/html/canvas/element/text-styles/2d.text.draw.baseline.ideographic.html I don't think these first two are about the same issue.

File: html/canvas/element/text-styles/META.yml [hunk 5]
```diff
@@ -22,3 +22,13 @@ links:
...
+        - test: 2d.text.draw.baseline.ideographic.html
+        - test: 2d.text.draw.space.collapse.end.html
```
Comment (foolip) [inline html/canvas/element/text-styles/META.yml:30]: These *.space.collapse.* tests do say "Space characters at the start of a line are collapsed (per CSS)" and while perhaps not identical to the problem in 2d.text.measure.width.space.html, let's keep them, it seems related enough.

**Review by foolip — CHANGES_REQUESTED** [review]

File: html/canvas/offscreen/text/META.yml (hunk 2 above)
Comment (foolip) [reply html/canvas/offscreen/text/META.yml:7]: Let's drop the baseline ones and keep *. space.collapse.* here too.

Comment (stephenmcgruer) [reply html/canvas/offscreen/text/META.yml]: Dropped:

```
        - test: 2d.text.draw.baseline.hanging.html
        - test: 2d.text.draw.baseline.ideographic.html
```

File: html/canvas/element/text-styles/META.yml (hunk 4 above)
Comment (stephenmcgruer) [reply html/canvas/element/text-styles/META.yml]: Dropped:

```
        - test: 2d.text.draw.baseline.hanging.html
        - test: 2d.text.draw.baseline.ideographic.html
```

**Review by foolip — APPROVED** [review]

**Pull request merged**